vector/src/main/AndroidManifest.xml

```diff
@@ -187,7 +187,6 @@
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value=".features.home.HomeActivity" />
         </activity>
-        <activity android:name=".features.debug.DebugMenuActivity" />
```

Comment:
this can now safely exist in only the debug sourceset as the debug implementation of `DebugNavigator` is the only place to reference the activity